ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -406,6 +408,21 @@ const triggerSet: TriggerSet<Data> = {
       },
       default: 'xsct',
     },
+    {
+      id: 'pangenesisFirstTower',
+      name: {
+        en: 'Pangenesis: First Towers',
+      },
+      type: 'select',
+      options: {
+        en: {
+          'Call Required Swaps Only': 'agnostic',
+          '0+2 (Conga Line/HRT)': 'not',
```

Comment:
Aren't they all conga line?

Reply:
> Aren't they all conga line?

Are they?  I don't know 1+2, and I've seen 0+2 referred to as conga line, but I don't know if that's a feature in common.  If it's ambiguous though, I can take it out.

Reply:
We do 1+2 with a conga line.  I think everybody does conga line or some version of flex to figure out who goes to which side when you both have the same debuff.  But, I think that's strat agnostic.